src/hypothesis/core.py

```diff
@@ -607,10 +607,11 @@ def wrapped_test(*arguments, **kwargs):
 
             random = get_random_for_wrapped_test(test, wrapped_test)
 
-            test_runner, search_strategy = process_arguments_to_given(
+            _processed_args = process_arguments_to_given(
                 wrapped_test, arguments, kwargs, generator_arguments,
                 generator_kwargs, argspec, test, settings
             )
+            arguments, kwargs, test_runner, search_strategy = _processed_args
```

Comment:
Yep, we blow past 79 characters if we assign it all on one line.

I’ll drop the leading underscore.